### DISCUSSION TITLE
Disabling headless mode during execution

### DISCUSSION THREAD

**Discussion by ApoNikol** [seleniumbase/SeleniumBase#3113] [Q&A] [answered]
Hello, i want to change the headless2 = True argument to False after a while, during the process of my script. How can i do it?
Do i have to use the websocket that appears when the headless instance is opened?( DevTools listening on ws://127.0.0.1:57484/devtools/browser/...). Thank you!

**Comment by mdmintz** [comment, marked as answer]
Parameters such as `headless2` are used during the initialization of the browser, and then can't be changed after the browser has already been launched.

**Reply by ApoNikol** [reply]
Alright,but Michael, is there any way to connect to this headless browser and interact with the gui?
Btw thank you you've been very supportive

**Reply by mdmintz** [reply]
`PyAutoGUI` requires a headed browser to work. That's not a problem on Linux due to the special virtual display that's included.
